mypy/checkexpr.py

```diff
@@ -53,7 +53,7 @@ def extract_refexpr_names(expr: RefExpr) -> Set[str]:
     MemberExpr."""
     output = set()  # type: Set[str]
     while expr.kind == MODULE_REF or expr.fullname is not None:
-        if expr.kind == MODULE_REF:
+        if expr.kind == MODULE_REF and expr.fullname is not None:
```

Comment:
Add a note either explaining why this might be `None` or telling that we don't know why (and we may want to investigate this in the future)? I have feeling that this being `None` might be due to a bug somewhere else.